tests/autorestart/test_container_autorestart.py

```diff
@@ -418,6 +419,12 @@ def run_test_on_single_container(duthost, container_name, service_name, tbinfo):
         logger.info("Restore auto-restart state of container '{}' to 'disabled'".format(container_name))
         duthost.shell("sudo config feature autorestart {} disabled".format(feature_name))
 
+    if container_name == "syncd":
```

Comment:
It's direct and effective. But can we make it general? In this test script, after each feature container was tested, it will post-check all containers' health in `check_all_critical_processes_status`, and this operation was executed in `wait_util`, so that can we put the `start-limit-hit` check here? I think it may have two advantages.
1. It will ignore whatever container testing triggered bgp or other services into start-limit-hit.
2. `wati_util` has a parameter `timeout`, it was controlled by static variable `POST_CHECK_THRESHOLD_SECS` and default is `360`s, it's enough to make bgp or other service state stable.

Reply:
Good suggestion, we can make it general if we add another `wait_until` with `check_all_critical_processes_status` with `timeout` of 100 sec in `postcheck_critical_processes_status`  before we check if `start-limit-hit` is reached. Because after we restart `start-limit-hit` we need to verify that all process are up, so initial check with `timeout` of 360 will still be needed. 

I suspect that bgp may reach `start-limit-hit` due to latest changes which made bgp container dependent from swss. 

Updated PR with appropriate changes.